modules/reference/pages/admin/users.adoc

```diff
@@ -1,10 +1,10 @@
 [[ref.webui.admin.users]]
 = Users
 
-To view and manage all users of the organization you are currently logged in to, click menu:Main Menu[Admin > Users] in the left navigation bar.
-The table lists user name, real name, organization and whether the user is organization or {productname} administrator.
+The menu:Admin[Users] section allows you to view and manage all users of the organization you are logged in to.
+Every user shows the username, real name, the organization they are associated with, and whether or not the user is an organization or or {productname} administrator.
```

Comment:
```suggestion
Every user shows the username, real name, the organization they are associated with, and whether the user is an organization or {productname} administrator.
```
do not forget that {productname} can expand to Uyuni. Writing "a  {productname} sth" would be problematic.